server/apps/users/views.py

```diff
@@ -131,3 +132,100 @@ def get(request):
         serializer = TodoSerializer(todos, many=True)
 
         return Response(serializer.data)
+
+    # Update to'do
+    @staticmethod
+    def put(request, **kwargs):
+        user = request.user
+        todo_id = request.data['todo_id']
+
+        # Get the to'do that user want to update from database and check if to'do in our database has current user as a editor
+        todo = Todo.objects.filter(id=todo_id).first()
+        editors = todo.editors.all()
+
+        # If current user is not an owner or editor for the to'do then don't process the request
+        if user != todo.owner and user not in editors:
+            return Response(OPERATION_NOT_ALLOWED)
```

Comment:
A better place to do this would be get_queryset which is one of the methods on view.